invenio_communities/templates/invenio_communities/mycommunities.html

```diff
@@ -8,7 +8,8 @@
 #}
 
 {%- set new_disabled_message = "To create a community your account must be verified for at least " + humanize.naturaldelta(config.COMMUNITIES_USER_CONFIRMED_SINCE) + ". If you want to speed up the process, you can contact our support team describing your case." -%}
-{% set can_create_community = can_user_create_community(current_user) %}
+{% set can_create_community = current_app.config.get(
```

Comment:
Since the check returns two values now, this would have to be unpacked in to

 ```suggestion
{% set can_create_community, can_create_error_message = current_app.config.get(
```

and the variable/message above can be removed